src/content/en/2021/capabilities.md

```diff
@@ -1,20 +1,234 @@
-authors: []
-reviewers: []
-analysts: []
+description: Capabilities give web apps access to hardware interfaces, enhance web-based productivity apps, and more.
+authors: [christianliebel]
+reviewers: [tomayac, hemanth]
+analysts: [tomayac]
 editors: []
 translators: []
 results: https://docs.google.com/spreadsheets/d/1b4moteB9EiLYkH1Ln9qfi1tnU-E4N2UQ87uayWytDKw/
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
+christianliebel_bio: Christian Liebel is a consultant at <a hreflang="en" href="https://thinktecture.com">Thinktecture</a>, supporting clients from various business areas in implementing great web applications. He is a Microsoft MVP for Developer Technologies, Google GDE for Web/Capabilities and Angular, and participates in the W3C Web Applications Working Group.
+featured_quote: Capabilities are new web platform APIs that unlock entirely new use cases for web applications.
+featured_stat_1: 9%
+featured_stat_label_1: Desktop web sites use the Web Share API
+featured_stat_2: 8.25%
+featured_stat_label_2: Mobile sites use the Async Clipboard API
+featured_stat_3: 11
+featured_stat_label_3: Mobile sites use Declarative Link Capturing
 ---
 
-## TODO
+## Capabilities
+Capabilities are new web platform APIs that unlock entirely new use cases for web applications. Those new APIs are essential for [Progressive Web Apps (PWA)](../pwa), a web-based application model. A PWA is a web app that users can install to their system. PWAs run even offline and launch quickly. To integrate with the underlying operating system, PWAs can only use web platform APIs. While browsers have already exposed some lower-level features to the web (e.g., geolocation, gamepad, or webcam access), many APIs were still missing or were clumsy to use (e.g., file system or clipboard access).
+
+The [Capabilities Project](https://www.chromium.org/teams/web-capabilities-fugu) (codename Fugu) is a joint effort by Microsoft, Intel, Google, and other Chromium contributors. It tries to bridge the gap between platform-specific applications and web apps by designing and implementing new powerful web platform APIs in a secure and privacy-preserving manner. As capabilities unlock more and more use cases, they lay the path for entire new application categories to finally make the shift to the web (e.g., IDEs, image editors, or office applications). Over the last two years, the focus for the Fugu team has been on capabilities for desktop productivity applications and hardware-related APIs. This chapter briefly introduces several new capabilities and analyzes how many different desktop and mobile websites use them. As capabilities are particularly interesting for app-like websites, their relative usage is comparatively low. This is why absolute website numbers are used in this chapter. For each capability, there will be a demo website or app that makes use of it.
```

Comment:
We've actually switched to using HTML links for external links so we can add a `hreflang="en"` so the translations get a little `(en)` symbol when a translation doesn't exist:

```suggestion
The <a hreflang="en" href="https://www.chromium.org/teams/web-capabilities-fugu">Capabilities Project</a> (codename Fugu) is a joint effort by Microsoft, Intel, Google, and other Chromium contributors. It tries to bridge the gap between platform-specific applications and web apps by designing and implementing new powerful web platform APIs in a secure and privacy-preserving manner. As capabilities unlock more and more use cases, they lay the path for entire new application categories to finally make the shift to the web (e.g., IDEs, image editors, or office applications). Over the last two years, the focus for the Fugu team has been on capabilities for desktop productivity applications and hardware-related APIs. This chapter briefly introduces several new capabilities and analyzes how many different desktop and mobile websites use them. As capabilities are particularly interesting for app-like websites, their relative usage is comparatively low. This is why absolute website numbers are used in this chapter. For each capability, there will be a demo website or app that makes use of it.
```

Note wikipedia and MDN links don't need this and can use markdown as we pick up the fact they have `en` in their URLs, so can continue to use markdown link syntax for them.

https://github.com/HTTPArchive/almanac.httparchive.org/blob/78ea8ea14f748a9f88d22cfcb48700eaf4bc8ced/src/static/css/almanac.css#L1559-L1587

Reply:
@tunetheweb How to deal with the `[Progressive Web Apps (PWA)](../pwa)` link? Does that need to be an HTML link too? I converted the other links (except the MDN ones).

Reply:
We leave internal links as they are. In the overly optimistic hope that they will be translated to every language 😁

Technically the link (since it is relative) is to the translated version, but in reality it will redirected to English most of the time, since a translated version will often not exist. 😞

Reply:
Cool, thanks!